website/blog/2020-06-25-wavemaker-openapi-postgrest-support.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "PostgREST OpenAPI support in WaveMaker"
+author: Nikhilesh K V
+---
+
+WaveMaker has added support for integration with [OpenAPIs](https://www.wavemaker.com/learn/blog/2020/04/21/wavemaker-openapi-import) in 10.4.  However, the OpenAPI definition document for REST APIS generated using [PostgREST](http://postgrest.org/en/v7.0.0/) looks different than a document generated using Spring. Support for such documents has been added in 10.5.
+<!-- truncate -->
+
+![OpenApi](/learn/assets/postGRESTlogo.png)
+
```

Comment:
In the introduction part, can we also mention, "As we are trying out more and more variations of OpenAPI docs from different sources of generation tools, our OpenAPI parser is evolving too. One such example is when we tested OpneAPI doc generated through PostgREST. In this blog we'll discuss what changes we have made in order to support OpenAPI docs generated from such source." 

This will set the context of the document.

Reply:
Also work in a reference to [Robustness principle](https://en.wikipedia.org/wiki/Robustness_principle)

What i mean to say is that in implementing a robust openAPI support, we tested various API across to be flexible in accepting many subtle variations how API is documented...